File: Readio/src/main/java/ninegle/Readio/tosspay/dto/TossPaymentConfirmResponseDto.java
package ninegle.Readio.tosspay.dto;

/**
 * @param status  결제 상태 "DONE" */

public record TossPaymentConfirmResponseDto(String orderId, String paymentKey, String status, String requestedAt,
											String approvedAt, int totalAmount) {
}

File: Readio/src/main/java/ninegle/Readio/tosspay/dto/TossPaymentConfirmResponseDto.kt
package ninegle.Readio.tosspay.dto

/**
 * @param status  결제 상태 "DONE"
 */
@JvmRecord
data class TossPaymentConfirmResponseDto(
	val orderId: String,
	val paymentKey: String,
	val status: String,
	val requestedAt: String,
	val approvedAt: String,
	val totalAmount: Int
)
